app/models/user.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      first_name: { type: DataTypes.STRING },
+      last_name: { type: DataTypes.STRING },
+      mail: { type: DataTypes.STRING },
```

Comment:
Careful here, you named it `mail` here, but you declared it as `email` in the migration file